msteams-platform/promote-app-adoption.md

```diff
@@ -102,7 +102,7 @@ To learn and understand how an IT admin can configure your app, see [Help IT adm
 
 Provide a user guide for your app and cover the following articles:
 
-* Installation instructions for your app from the Teams store.
+* Instructions to install your app from the [Teams app store](https://appsource.microsoft.com/marketplace/apps?product=office%3Bteams).
```

Comment:
Also, why link all instances to the destination? First instance should definitely have a CTA but if all mentions have links then it is difficult to read.

Reply:
We believe the context here revolves around apps available in Teams client store. If we give a link to AppSource it would change the context to add instructions for app from AppSource and not the Teams client store. Request you to please look into it.

Reply:
AFAIK, we can't link right into the client's store. We the only way to link is to the AppSource on the web. I think the latter is OK since any app link on AppSource site is a so-called deep link to the app in the Store. 
Also, what if we leave this text as is without linking it to any app acquisition UI?

Reply:
Yes @ashishguptaiitb, I agree we can leave the text as is without linking.